meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging/am62xx-evm/lmp.cfg

```diff
@@ -6,3 +6,4 @@ CONFIG_ENV_FAT_DEVICE_AND_PART="1:1"
 CONFIG_BOOTCOUNT_LIMIT=y
 CONFIG_BOOTCOUNT_BOOTLIMIT=3
 CONFIG_BOOTCOUNT_ENV=y
+CONFIG_SYS_BOOTM_LEN=0x4000000
```

Comment:
Can you say that this change is needed because of the u-boot version update done in the ti layer?

Reply:
In the commit message is fine.

Reply:
offtopic: I think we might also set this by default in lmp-common.cfg when it's merged, as technically Linux FIT is usually the same size across all ARM64 boards

Reply:
added the notes on the commit message